tooling/nargo_fmt/tests/expected/assert.nr

```diff
@@ -1,4 +1,5 @@
 fn main(x: Field) {
     assert(x == 0, "with a message");
     assert_eq(x, 1);
+    assert_eq(x, message);
```

Comment:
Small change, the rest of your PR is about the third message arg but you're placing the `message` variable in the 2 arg. This is equivalent to `assert(x == message)`